frontend/src/styles/CreateMoment.css

```diff
@@ -1,31 +1,227 @@
+@import url('https://fonts.googleapis.com/css?family=Teko');
+
 .CreateMoment {
+    text-align: center;
+    font-size: 30px;
     margin-top: 20px;
-    margin-left: 20px;
+    margin-left: 40px;
     margin-right: auto;
     padding-left: 0px;
     padding-top: 0px;
     height: 20%;
 }
 
+div.createMoment-header{
+  text-align: center;
+  margin-top: 20px;
+  padding-bottom: 40px;
+}
+
+div.createMoment-header img{
+  text-align: center;
+  width: 420px;
+}
+
+div.createMoment-description {
+  display: block;
+  position: center;
+  text-align: center;
+  font-family: 'Teko';
+  color: #1DE28F;
+  font-size: 48px;
+}
 .content-container {
+    display: block;
+    position: center;
+    margin-top: 30px;
     margin-bottom: 20px;
-}
 
+}
 .label {
     margin-right: 40px;
 }
 
-.title { 
-    width: 300px;
-}
-
 .textarea {
     padding-left: 10px;
+
 }
 
 .button {
-    background-color: rgb(218, 222, 224);
-    font-size: 16px;
-    border-radius: 12px;
-    border-color: black;
-}
\ No newline at end of file
+    cursor: pointer;
+    background-color: rgb(255, 255, 255);
+    font-size: 26px;
+    font-family: 'Teko';
+    width: 90px;
+    border-radius: 2px;
+}
+.button:hover{
+    border-width: 2px;
+    border-color: #1DE28F;
+}
+/*
+.upload-btn-wrapper input[type=file] {
+  font-size: 100px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  opacity: 0;
+}
+*/
+.inputfile {
+  cursor: pointer;
+	width: 0.1px;
+	height: 0.01px;
+	overflow: hidden;
+	position: absolute;
+	z-index: -1;
+  background-color: white;
```

Comment:
sometimes you write #FFFFFF and sometimes white. choose one and make it consistent